### PR TITLE
Update style dictionary to v4

### PR DESCRIPTION
**Pull request by akomiqaia**
## Checklist before requesting a review

- [ ] I have applied `percy` label for o-[COMPONENT] or `chromatic` label for o3-[COMPONENT] on my PR before merging and after review. Find more details in [CONTRIBUTING.md](https://github.com/Financial-Times/origami/blob/main/CONTRIBUTING.md#pull-requests-and-visual-regression-tests)
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

File: components/o3-editorial-typography/src/css/brands/professional.css [hunk 1]
```diff
@@ -1,3 +0,0 @@
-@import '@financial-times/o3-foundation/css/professional.css';
```
Comment (frshwtr) [inline components/o3-editorial-typography/src/css/brands/professional.css:1]: Why were these removed?

File: libraries/o3-tooling-token/build/core/_variables.js [hunk 2]
```diff
@@ -172,7 +172,7 @@ export default {
 },
 	"o3-color-palette-transparrent": {
 		"shortName": "transparrent",
-		"value": "#ffffff00",
+		"value": "#ffffffff",
```
Comment (frshwtr) [inline libraries/o3-tooling-token/build/core/_variables.js:175]: This looks incorrect, it's changed transparrent to solid.

File: libraries/o3-tooling-token/build/core/_variables.js [hunk 3]
```diff
@@ -1616,7 +1616,7 @@ export default {
 },
 	"o3-spacing-4xl": {
 		"shortName": "4xl",
-		"value": "6rem",
+		"value": "96px",
```
Comment (frshwtr) [inline libraries/o3-tooling-token/build/core/_variables.js:1619]: I can't remember what we decided, but were we changing these from `rem` to `px`?

**Comment by frshwtr** [comment]
I think this LGTM with some comments.

I think you mentioned you wanted @notlee to take a look at this also. I'll let Lee have the approving review.

File: libraries/o3-tooling-token/build/core/_variables.js (hunk 3 above)
Comment (notlee) [reply libraries/o3-tooling-token/build/core/_variables.js:1619]: Good spot. This should match the CSS token value, whether px or rem. They are currently different for some reason.

Comment (notlee) [reply libraries/o3-tooling-token/build/core/_variables.js:1619]: I think we should actually switch to `px` for all units to facilitate initial rollout as there are a bunch of ft.com projects which set a lower than 16px root font size 🙃 That doesn't need to be this PR

**Comment by notlee** [comment]
No comments beyond Ben's :)

File: components/o3-editorial-typography/src/css/brands/professional.css (hunk 1 above)
Comment (akomiqaia) [reply components/o3-editorial-typography/src/css/brands/professional.css:1]: i don't professional supports editorial-typography.

**Comment by notlee** [comment]
Sorry, on closer look I'm seeing issues. E.g. type now / before:
Screenshot 2024-09-06 at 12 29 32

Could you ensure a visual regression test is run for this PR before merging? Even if that includes fixing any issue there first

**Comment by notlee** [comment]
@akomiqaia ah hold that thought, my token build failed

**Review by frshwtr — APPROVED** [review]
